notebook/static/tree/js/notebooklist.js

```diff
@@ -935,6 +951,22 @@ define([
         });
     };
 
+    NotebookList.prototype.view_selected = function() {
+        var that = this;
+        that.selected.forEach(function(item) {
+            var item_path = item.path;
+            var w = window.open(utils.url_path_join('/files', item_path));
```

Comment:
I think we need a `url_path_encode` on item_path, plus use `IPython._target` for the window target. See a few of the other URL-building and window-opening bits on this page.